CONTRIBUTING.md

```diff
@@ -25,19 +19,20 @@ Please make sure to read and observe our [Contributor Code of Conduct](./CODE_OF
 ## Contributing Guidelines
 
 ### Getting Started
-`Good first issues` are a great way to start contributing to the project and get familiar with the codebase.
-Here's how to find them:
-
-1. Visit the "[Issues](https://github.com/devopness/devopness/issues)" tab on the main [repository](https://github.com/devopness/devopness)
-2. Use the "Labels" filter and select "[Good First Issue](https://github.com/devopness/devopness/labels/good%20first%20issue)" to see a list of beginner-friendly tasks
-3. Choose an issue that interests you
-
-### Forking this repository
-1. [Fork the repository](https://github.com/devopness/devopness/fork) and start working on it
-2. To become familiar working with forks, please read and follow the [standard fork based workflow](https://gist.github.com/Chaser324/ce0505fbed06b947d962), which could be summarized with the following steps:
-  - Fork Devopness repository
-  - Create a branch
-  - When the changes in a branch are tested, open a pull request from your fork
+
+1. Make sure you can access [Devopness projects](https://app.devopness.com/projects)
+2. [Fork the repository](https://github.com/devopness/devopness/fork) and start working on it
+  - To become familiar working with forks, please read and follow the [standard fork based workflow](https://gist.github.com/Chaser324/ce0505fbed06b947d962), which could be summarized with the following steps:
+    - Fork Devopness repository
+    - Create a branch
+    - When the changes in a branch are tested, open a pull request from your fork 
+4. [Report a bug 🐛 or submit new ideas 🆕 💡](https://github.com/devopness/devopness/issues/new/choose)
+5.❓ Ask questions and help other Devopness users with open [discussions](https://github.com/devopness/devopness/discussions)
+6. Find a "[Good First Issue](https://github.com/devopness/devopness/labels/good%20first%20issue)"
```

Comment:
```suggestion
1. [Fork the repository](https://github.com/devopness/devopness/fork) and start working on it
  - To become familiar working with forks, please read and follow the [standard fork based workflow](https://gist.github.com/Chaser324/ce0505fbed06b947d962), which could be summarized with the following steps:
    - Fork Devopness repository
    - Create a branch
    - When the changes in a branch are tested, open a pull request from your fork 
2. [Report a bug 🐛 or submit new ideas 🆕 💡](https://github.com/devopness/devopness/issues/new/choose)
3.❓ Ask questions and help other Devopness users with open [discussions](https://github.com/devopness/devopness/discussions)
4. Find a "[Good First Issue](https://github.com/devopness/devopness/labels/good%20first%20issue)"
```

Complementing the suggestion made in [LINK](#), here the citation to Devopness APP would be removed